pkgs/top-level/release-cuda.nix

```diff
@@ -0,0 +1,55 @@
+/*
+    Test CUDA packages.
+
+    This release file will not be tested on hydra.nixos.org
+    because it requires unfree software.
+
+    Test for example like this:
+
+        $ hydra-eval-jobs pkgs/top-level/release-cuda.nix --option restrict-eval false -I foo=. --arg nixpkgs '{ outPath = ./.; revCount = 0; shortRev = "aabbcc"; }'
```

Comment:
@samuela @SomeoneSerge could you run this also inside a say `$ nix shell hydra`. Basically, the supported cudnn versions need to be fixed and I'm going to need a bit of help there. Likely the download urls are incorrect there as well.

Reply:
yep, should be fixed now. I need some help with choosing the default cudnn versions.